test/NuGet.Core.Tests/NuGet.Build.Tasks.Console.Test/MSBuildStaticGraphRestoreTests.cs

```diff
@@ -125,8 +127,34 @@ public void GetPackageDownloads_WhenNotExactVersion_ThrowsException(string versi
                 {
                     var _ = MSBuildStaticGraphRestore.GetPackageDownloads(project).ToList();
                 };
+                string expectedMessage = string.Format(Strings.Error_PackageDownload_OnlyExactVersionsAreAllowed, expected ?? version, packageName);
+                act.Should().Throw<ArgumentException>().WithMessage(expectedMessage);
+            }
+        }
+
+        [Fact]
+        public void GetPackageDownloads_NoVersion_ThrowsException()
+        {
+            string packageName = "PackageA";
+            using (var testDirectory = TestDirectory.Create())
+            {
+                var project = new MockMSBuildProject(testDirectory)
+                {
+                    Items = new Dictionary<string, IList<IMSBuildItem>>
+                    {
+                        ["PackageDownload"] = new List<IMSBuildItem>
+                        {
+                            new MSBuildItem(packageName, new Dictionary<string, string> { ["Version"] = null }),
+                        }
+                    }
+                };
+
+                Action action = () =>
+                {
+                    var _ = MSBuildStaticGraphRestore.GetPackageDownloads(project).ToList();
+                };
 
-                act.Should().Throw<ArgumentException>().WithMessage($"'{expected ?? VersionRange.Parse(version).OriginalString}' is not an exact version like '[1.0.0]'. Only exact versions are allowed with PackageDownload.");
+                action.Should().Throw<ArgumentException>().WithMessage(string.Format(CultureInfo.CurrentCulture, Strings.Error_PackageDownload_NoVersion, packageName));
```

Comment:
Nice, you're also contributing towards making our tests localized instead of hardcoded to `en-us`!